Point[2] points